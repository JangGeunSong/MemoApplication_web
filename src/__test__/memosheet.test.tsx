import React from 'react'
import { render, fireEvent } from '@testing-library/react'
import MemoSheet from '../components/memosheets'

describe('Memosheet test', () => {
    const memosheet = render(<MemoSheet />)

    it('render test', () => {
        expect(memosheet.container).toMatchSnapshot();
    })
})
