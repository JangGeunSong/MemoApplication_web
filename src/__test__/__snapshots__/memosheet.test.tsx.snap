// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Memosheet test render test 1`] = `
<div>
  <div
    class="memo-sheet"
  >
    <div
      class="memo-sheet"
    />
  </div>
</div>
`;
